flycheck.el

```diff
@@ -3942,8 +3967,10 @@ Return a list with the contents of the table cell."
                    column 'flycheck-error-list-column-number)
                   (flycheck-error-list-make-cell
                    (symbol-name (flycheck-error-level error)) level-face)
-                  (flycheck-error-list-make-cell
-                   id-str 'flycheck-error-list-id id-str)
+                  ;; Error ID has a button to explain the error
+                  (list id-str
+                        'type 'flycheck-error-list-explain-error
+                        'face 'flycheck-error-list-id)
```

Comment:
Maybe we could extend -error-list-make-cell? Or just drop the full -error-list-make-cell function?